libraries/AP_HAL_ChibiOS/hwdef/BeastH7v2/hwdef.dat

```diff
@@ -0,0 +1,42 @@
+# hw definition file for processing by chibios_pins.py
+# for iFlight Beast H7 v2 hardware, based on Beast H7 v1
+# thanks to betaflight for pin information
+
+include ../BeastH7/hwdef.dat
+undef APJ_BOARD_ID
+undef PD15 PB10 PB11 
+undef HAL_SERIAL3_PROTOCOL
+undef IMU
+undef BARO
+
+# board ID for firmware load
+APJ_BOARD_ID 1056
+
+# only one I2C bus
+I2C_ORDER I2C2
+
+# order of UARTs (and USB)
+# This setup disables USART3 to enable the I2C bus for compass
+SERIAL_ORDER OTG1 USART1 USART2 EMPTY UART4 EMPTY EMPTY UART7
+# Alternative configuration with USART3
+# SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 EMPTY EMPTY UART7
+
+# Buzzer - DMA timer channel use by LEDs
+PD2 BUZZER OUTPUT GPIO(80) LOW
+
+# I2C2 for compass. These pins can also be used as USART3
+PB10 I2C2_SCL I2C2 
+PB11 I2C2_SDA I2C2 
```

Comment:
You should add an ALT configuration to allow these to be used as a UART. Tridge thinks this should work - just put ALT(1) on the end of the alternate pin definitions

Reply:
fixed, but not tested with real hardware yet